src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -12,14 +12,12 @@ import PropertyResourceTemplate from './PropertyResourceTemplate'
 import lookupConfig from '../../../static/spoofedFilesFromServer/fromSinopiaServer/lookupConfig.json'
 import { getLD, setItems, removeAllContent } from '../../actions/index'
 import { getResourceTemplate } from '../../sinopiaServer'
+import {resourceToName} from "../../Utilities";
```

Comment:
totally ok to ignore ... all other curly braces nearby have a space.  :-P

Reply:
I guess people like spaces in their curly braces... :)